pkg/destroy/gcp/backendservice.go

```diff
@@ -2,69 +2,100 @@ package gcp
 
 import (
 	"context"
+	"fmt"
 
-	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 	"google.golang.org/api/compute/v1"
 	"google.golang.org/api/googleapi"
 
 	"github.com/openshift/installer/pkg/types/gcp"
 )
 
-func (o *ClusterUninstaller) listBackendServices(ctx context.Context) ([]cloudResource, error) {
-	return o.listBackendServicesWithFilter(ctx, "items(name),nextPageToken", o.clusterIDFilter(), nil)
+func (o *ClusterUninstaller) listBackendServices(ctx context.Context, scope resourceScope) ([]cloudResource, error) {
+	return o.listBackendServicesWithFilter(ctx, "items(name),nextPageToken", o.clusterIDFilter(), nil, scope)
+}
+
+func createBackendResources(filterFunc func(*compute.BackendService) bool, list *compute.BackendServiceList) []cloudResource {
```

Comment:
should this be something like `findBackendResources` instead of `createBackendResources`?

Reply:
@patrickdillon I am changing the function name but this function is not finding backend resources this is creating cloudResources (the internal struct for deletion).